_templates/community.html

```diff
@@ -104,6 +104,12 @@ <h3 id="canada"><img src="/img/flags/CA.png" alt="Canadian flag">Canada</h3>
   </div>
 </div>
 <div>
+ +  <h3 id="colombia"><img src="/img/flags/CO.png" alt="Bandera de Colombia / Colombian Flag">Colombia</h3>
```

Comment:
@santy2001 Could you please revise this to `Colombian flag` so that it matches the schema of the other ALT tags?